resume.md

```diff
@@ -20,12 +20,14 @@ cvfont: "lato"
 
 # Objective
 
-After 7 years working with Customer Relations Management applications (CRMs), I am eagar to transition into another field. I've put considerable effort into 
-learning Python and it quickly became my favorite languange. I am seeking opportunities in scientific applications, data science, or Python web development. \newline
+After 7 years of software development focused on internal business logistics, I am eager to explore other realms.
+Over the last {time}, I invested in learning Python in support of this objective, and I am now seeking
+opportunities in research software engineering, data science, and Python web development. \newline
+
```

Comment:
Much better than what I have. I suppose if they want more details about 'business logistics' and what CRMs are I can provide that in an interview.

Reply:
I think that's a great idea, but don't include "since I've not used it professionally". I would define your open source contributions as "professional" even though you didn't get paid.

Maybe you could find a way to work in "learning Python through XYZ course openly on GitHub". I'm struggling to see a smooth way to do it at the moment :)